templates/frontend/components/header.tpl

```diff
@@ -91,6 +96,8 @@
 				<nav class="pkp_navigation_user_wrapper" id="navigationUserWrapper" aria-label="{translate|escape key="common.navigation.user"}">
 					{load_menu name="user" id="navigationUser" ulClass="pkp_navigation_user" liClass="profile"}
 				</nav>
+				</div>
+
```

Comment:
The HTML code between the opening and closing divs here needs to be indented one more space.